### ISSUE TITLE
Support alias and plugins

### ISSUE THREAD

**Issue by danielroe** [nuxt/vite#3]
```
 ERROR  Client compiled with errors: The following dependencies are imported but could not be resolved:

  nuxt_plugin_plugin_1aac32db (imported by /my/repo/.nuxt/index.js)
  nuxt_plugin_pluginclient_8259ce52 (imported by /my/repo/.nuxt/index.js)
  nuxt_plugin_pluginserver_248c8a5f (imported by /my/repo/.nuxt/index.js)
  nuxt_plugin_plugin_720efac8 (imported by /my/repo/.nuxt/index.js)
  nuxt_plugin_pluginserver_722e082b (imported by /my/repo/.nuxt/index.js)
  nuxt_plugin_pluginclient_0c7496a3 (imported by /my/repo/.nuxt/index.js)
  nuxt_plugin_meta_3875608a (imported by /my/repo/.nuxt/index.js)
```

**Comment by pi0** [comment]
We need to implement alias support (https://github.com/nuxt/nuxt.js/blob/59fa54ffeb86400920c3b5e19d50d3e4594812fe/packages/webpack/src/config/base.js#L283)

**Comment by pi0** [comment]
Fixed in 0.0.3
